yarn.lock

```diff
@@ -5757,6 +5757,14 @@ axios@^0.21.1:
   dependencies:
     follow-redirects "^1.14.0"
 
+axios@^0.27.2:
+  version "0.27.2"
+  resolved "https://registry.yarnpkg.com/axios/-/axios-0.27.2.tgz#207658cc8621606e586c85db4b41a750e756d972"
+  integrity sha512-t+yRIyySRTp/wua5xEr+z1q60QmLq8ABsS5O9Me1AsE5dfKqgnCFzwiCZZ/cGNd1lq4/7akDWMxdhVlucjmnOQ==
+  dependencies:
+    follow-redirects "^1.14.9"
+    form-data "^4.0.0"
+
 axios@~0.21.4:
```

Comment:
sorry last thing: shouldn't axios `0.21.4` be removed now?

Reply:
It looks like utilities is still using the old version:

<img width="419" alt="image" src="https://user-images.githubusercontent.com/1641214/167912238-4cb72358-32dc-44fa-a08b-256e834684a1.png">

Reply:
For some reason `services-store` is still using the incorrect `utilities` version (which was still using `axios@0.21.4`).  @jeffchew Didn't you push a fix for this recently?

Reply:
@jeffchew Looks to be unrelated to vendor – `services-store` correctly bumps the `services` version but not [`utilties`](https://github.com/carbon-design-system/carbon-for-ibm-dotcom/blob/main/packages/services-store/package.json#L43). Not sure what's happening, will investigate further. In the meantime I've manually bumped the version to latest.